app/views/posts/index.html.erb

```diff
@@ -15,6 +15,7 @@
         <div class="card-body">
           <h5 class="card-title"><%= post.title %></h5>
           <p class="card-text"><%= post.text.truncate(100) %></p>
+          <% if post.tag.name!="" %> <div class= "badge badge-pill badge-info"> <%=  post.tag.name %> </div><br> <br><% end %>
```

Comment:
![image](https://github.com/user-attachments/assets/e36f0ecf-3086-4443-a6ca-3370f5403260)
Getting error from this line when posts do not exist

Reply:
Sorry, I'm having trouble replicating this issue. So, this error is occurring when there are no posts in the database and you enter the post index route?  When I try that, I have no such issue.  I'm not sure why it would happen in that case because I believe that code is inside a for loop that only enters if there are any posts in the database.